src/modules/NtpClient.rb

```diff
@@ -92,8 +92,9 @@ def main
       # The interval of synchronization in minutes.
       @sync_interval = DEFAULT_SYNC_INTERVAL
 
-      # Service name of the NTP daemon
+      # Service names of the NTP daemon
       @service_name = "chronyd"
+      @wait_service_name = "chrony-wait"
```

Comment:
can you please add here comment why it is needed, ideally with reference to bugzilla.